modules/liveIntentIdSystem.js

```diff
@@ -8,9 +8,10 @@ import * as utils from '../src/utils.js';
 import { triggerPixel } from '../src/utils.js';
 import { ajaxBuilder } from '../src/ajax.js';
 import { submodule } from '../src/hook.js';
-import { LiveConnect } from 'live-connect-js/cjs/live-connect.js';
+import { LiveConnect } from 'live-connect-js/esm/initializer.js';
 import { gdprDataHandler, uspDataHandler } from '../src/adapterManager.js';
 import { getStorageManager } from '../src/storageManager.js';
+import { MinimalLiveConnect } from 'live-connect-js/esm/minimal-live-connect.js';
```

Comment:
Will this result in both libraries (LiveConnect and MinimalLiveConnect) being added to the bundle?

Reply:
Hey @msm0504 well, the bundle is generated by webpack, so no.

If this were pieced together without any treeshaking, then yes, both sources would have been added, however, the trick here is that [at some point](https://github.com/prebid/Prebid.js/pull/6016/files#diff-489289275f4ee5264024b54c9c851e6c57684fd2d0bccab91d8b9ccef150e235R125), there's an evaluation of a boolean which is either true or false, and that's how webpack will know which import will be used. Anything unused will be automatically removed from the bundle.